python/ray/serve/_private/replica.py

```diff
@@ -664,7 +669,10 @@ async def prepare_for_shutdown(self):
         while True:
             # Sleep first because we want to make sure all the routers receive
             # the notification to remove this replica first.
-            await asyncio.sleep(self.deployment_config.graceful_shutdown_wait_loop_s)
+            if self.deployment_config:
```

Comment:
why's this necessary?

Reply:
Eyeballing the log and find this error(might not related to the release test), because self.deployment_config is initialized to be None in the constructor, if no `start` to be called, and then being called to stop, it might cause None type error.

Reply:
got it. we should still use some short default here else this will basically busy-spin